misc/acrn-config/launch_config/com.py

```diff
@@ -383,8 +383,12 @@ def set_dm_pt(names, sel, vmid, config):
             sel.bdf["bluetooth"][vmid][3:5], sel.bdf["bluetooth"][vmid][6:7]), file=config)
 
     if sel.bdf['wifi'][vmid] and sel.slot['wifi'][vmid]:
-        print("   -s {},passthru,{}/{}/{},keep_gsi \\".format(sel.slot["wifi"][vmid], sel.bdf["wifi"][vmid][0:2], \
-            sel.bdf["wifi"][vmid][3:5], sel.bdf["wifi"][vmid][6:7]), file=config)
+        if uos_type == "ANDROID":
```

Comment:
Actually, the flag doesn't rely on uos type, but the driver implementation.
Does linux guest wifi dirver support msi?

Reply:
When the flag relay on the uos driver implementation,  use uos type as the condition should be ok.
The guest should be customized from user,  it maybe support msi or not. If the guest not support the flag, once this flag is set, there is some risk to start the guest successfully.
As such scenario, use Android uos type is the proper condition to match the qa and the user requirement.